test/e2e/README.md

```diff
@@ -71,6 +71,12 @@ Multiple testnets can be run with the `run-multiple.sh` script:
 ./run-multiple.sh networks/generated/gen-group3-*.toml
 ```
 
+To save the execution files specify the `SAVE_EXECUTION` variable:
+
+```sh
+ SAVE_EXECUTION=1 ./run-multiple.sh networks/generated/gen-group3-*.toml
+ ```
+
```

Comment:
I would add here a section to the introduced section below, which explains what is saved, how, and where.

Reply:
OK, I'll add that